### DISCUSSION TITLE
react-dropdown-tree-select

### DISCUSSION THREAD

**Discussion by muhammadmadni-ivl** [dowjones/react-dropdown-tree-select#524] [Q&A]
I want some hint that how should I tweak the component for chakra-ui, 
![image](https://user-images.githubusercontent.com/78340648/135876522-4914466a-9e79-41c5-984c-950090fd981c.png)
if see that there is (x) at the end of each filtered value and it's working as expected, there is no plus button to expand further values under each selection.

**Comment by mrchief** [comment]
@muhammadmadni-ivl mind creating a CodeSandbox for this?
